src/test/data/JsonMemeBookStorageTest/invalidAndValidMemeMemeBook.json

```diff
@@ -1,9 +1,9 @@
 {
   "memes": [ {
     "name": "Valid Person",
-    "address": "4th street"
+    "description": "4th street"
   }, {
-    "name": "Person With Invalid Address",
-    "address": ""
+    "name": "Person With Invalid Description",
```

Comment:
So with that change, this would change as well, there won't be an invalid description, so change this to an invalid name.